x/feemarket/keeper/eip1559.go

```diff
@@ -43,7 +43,7 @@ func (k Keeper) CalculateBaseFee(ctx sdk.Context) *big.Int {
 
 	parentGasTargetBig := new(big.Int).Div(gasLimit, new(big.Int).SetUint64(uint64(params.ElasticityMultiplier)))
 	if !parentGasTargetBig.IsUint64() {
-		return new(big.Int).SetInt64(params.InitialBaseFee)
+		return nil
```

Comment:
why this change?

Reply:
same here, does it make sense to use `params.InitialBaseFee` when the parameter may be wrongly set?